packages/extension-browser/src/devtools/panel/views/pages/config/TargetBrowsers.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import { FormEvent } from 'react';
+import browserslist = require('browserslist');
+
+import { getMessage } from '../../../utils/i18n';
+
+import ConfigLabel from './ConfigLabel';
+import ConfigSection from './ConfigSection';
+import Example from './Example';
+import ValidatedInput from './ValidatedInput';
+
+type Props = {
+    query?: string;
+    onChange: (query?: string) => void;
+};
+
+const placeholder = '> 1%, IE 10';
+
+/** Check if a user's custom `browserslist` query is valid, notifying them if it is not. */
+const validate = (value?: string): string => {
+    try {
+        browserslist(value);
+
+        return '';
+    } catch (e) {
+        // Report errors, stripping messages about "old" browserslist since the user won't have control over that.
+        return e.message.replace(' Maybe you are using old Browserslist or made typo in query.', '');
```

Comment:
Maybe we should replace this error saying something like:

> Maybe you made a typo in your query or are using a syntax not supported by browserslist vXXX

Having an empty error message is not very user friendly.

Reply:
This doesn't create an empty error message, it just strips the suffix.

Reply:
Great. Can you put a comment with an examination le of the message that will be modified? With the current one I was thinking the whole message was going to get removed.